relay-server/src/metrics_extraction/snapshots/relay_server__metrics_extraction__event__tests__extract_span_metrics.snap

```diff
@@ -1468,47 +891,6 @@ expression: metrics
             "span.op": "ui.load",
         },
     },
-    Bucket {
-        timestamp: UnixTimestamp(1695255136),
-        width: 0,
-        name: "d:spans/exclusive_time@millisecond",
-        value: Distribution(
-            [
-                1668.516159,
-            ],
-        ),
-        tags: {
-            "environment": "fake_environment",
-            "span.category": "app",
-            "span.description": "Cold Start",
-            "span.group": "2d675185edfeb30c",
-            "span.op": "app.start.cold",
-            "span.status_code": "200",
-            "transaction": "gEt /api/:version/users/",
-            "transaction.method": "POST",
-            "transaction.op": "myop",
-        },
-    },
-    Bucket {
-        timestamp: UnixTimestamp(1695255136),
-        width: 0,
-        name: "d:spans/exclusive_time_light@millisecond",
-        value: Distribution(
-            [
-                1668.516159,
-            ],
-        ),
-        tags: {
-            "environment": "fake_environment",
-            "span.category": "app",
-            "span.description": "Cold Start",
-            "span.group": "2d675185edfeb30c",
-            "span.op": "app.start.cold",
-            "span.status_code": "200",
-            "transaction.method": "POST",
-            "transaction.op": "myop",
-        },
-    },
```

Comment:
`app.start.*` metrics are only extracted for mobile SDKs, but there's no SDK info on the event in this test case. I figured we can omit these test inputs, because there's already a separate mobile test case.